lib/puppet/functions/extlib/cidr_to_broadcast.rb

```diff
@@ -0,0 +1,18 @@
+# Imported by Tim 'bastelfreak' Meusel into voxpupuli/extlib because Yelp/netstdlib got abandoned
+#
+# @summary Converts an CIDR address of the form 192.168.0.1/24 into its broadcast address
+#
+Puppet::Functions.create_function(:'extlib::cidr_to_broadcast') do
+  # @param ip IPv6 or IPv4 address in CIDR notation
```

Comment:
IPv6 doesn't have the concept of broadcast so I would reject it.

Reply:
dropped IPv6